wbooks/src/App/LoginPage/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { Redirect, Link } from 'react-router-dom';
+import LoginForm from './LoginForm';
+import { Auth } from '../Auth/authentication';
+import './styles.css';
+
+export default class LoginPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      redirect : false,
+    }
+  }
+
+  redirect = () => {
+    this.setState({
+      redirect: true
+    });
+  };
+
+  render() {
+    if(this.state.redirect || Auth.isLoggedIn()) {
+      return <Redirect to="/" />
+    }
+    return (
+      <div className="login-page">
+        <div className="wb_logo" ></div>
+        <LoginForm redirect={this.redirect} />
```

Comment:
We use a convention for callbacks like this.
We call the handler `handleEvent` and the received prop `onEvent`.

So here you can rename `redirect` to `handleRedirect` and then pass it like `onRedirect={this.handleRedirect}`